catalog/admin/admin-poller-service/src/test/java/org/codice/ddf/catalog/admin/poller/AdminPollerTest.java

```diff
@@ -183,11 +183,11 @@ protected AdminSourceHelper getHelper() {
                 // Mock out the metatypes
```

Comment:
For the record, I was able to pull the code out of the helper class and add a unit test that way. Not a difficult refactor and it'll make sure no one breaks the filter generator accidentally in the future.